evaluation/poets/scripts/verses_2om_3om_2instances.sh

```diff
@@ -1,5 +1,5 @@
 # verses with 2 or more, 3 or more, exactly 2 instances of light.
-INPUT=${INPUT:-$PATH_TOP/evaluation/scripts/input/genesis}
-grep -c 'light.*light' ${INPUT} 
-grep -c 'light.*light.*light' ${INPUT}
-grep 'light.*light' ${INPUT} | grep -vc 'light.*light.*light' 
+INPUT=${INPUT:-$PASH_TOP/evaluation/scripts/input/poets/genesis}
+grep -c 'light.*light' ${INPUT} | paste -sd+ | bc
```

Comment:
If possible, let's keep the original version of the scripts and put the alternative version (that doesn't have the issue) in a comment. The issue that happens now is solvable easily by modifying the grep annotation.

Reply:
Commenting out the `paste -sd+ | bc` should ideally happen for all `grep -c` occurences.